bitnami/kube-prometheus/Chart.yaml

```
@@ -51,4 +51,4 @@ maintainers:
 name: kube-prometheus
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/kube-prometheus
-version: 11.2.11
+version: 11.2.12
```

Please bump minor version instead given you're adding a new parameter:
```suggestion
version: 11.3.0
```